ui/components/ui/error-message/index.scss

```diff
@@ -19,4 +19,18 @@
   &__text {
     overflow: auto;
   }
+
+  &__link {
+    text-decoration: none;
+    cursor: pointer;
```

Comment:
Now that the link is an `<a>` element, it already has a pointer cursor. This is not needed.

```suggestion
```